### PR TITLE
Fix for point in time

### PR DESCRIPTION
**Pull request by Arpit-Bandejiya**
### Description
fixes the point in time functionality
[Describe what this change achieves]

### Fix Issue
[#4187](https://github.com/opensearch-project/OpenSearch/issues/4187)

### Testing
- [x] New functionality includes testing

[Describe how this change was tested]

---
By submitting this pull request, I confirm that my contribution is made under the terms of the Apache 2.0 license.
For more information on following Developer Certificate of Origin and signing off your commits, please check [here](https://github.com/opensearch-project/OpenSearch/blob/main/CONTRIBUTING.md#developer-certificate-of-origin).

### CODE REVIEW

File: osbenchmark/worker_coordinator/runner.py [hunk 1]
```diff
@@ -1925,9 +1937,11 @@ class Composite(Runner):
     def __init__(self, *args, **kwargs):
         super().__init__(*args, **kwargs)
         self.supported_op_types = [
-            "open-point-in-time",
-            "close-point-in-time",
+            "create-point-in-time",
+            "delete-point-in-time",
+            "get-point-in-time",
             "search",
+            "paginated-search",
```
Comment (bharath-techie) [inline osbenchmark/worker_coordinator/runner.py:1944]: Where it the code for this ?

Comment (Arpit-Bandejiya) [reply osbenchmark/worker_coordinator/runner.py:1944]: The code is already present in the code base

File: osbenchmark/worker_coordinator/runner.py [hunk 2]
```diff
@@ -1848,32 +1849,43 @@ def __repr__(self, *args, **kwargs):
...
+        }
+        await opensearch.delete_point_in_time(body=body, params=request_params, headers=None)
         CompositeContext.remove(pit_op)
 
     def __repr__(self, *args, **kwargs):
-        return "close-point-in-time"
+        return "delete-point-in-time"
+
+
+class GetPointInTime(Runner):
```
Comment (dhruv16dhr) [inline osbenchmark/worker_coordinator/runner.py:1882]: Should we make it list all? This will be in-sync with the changes being made in opensearch backend

Comment (Arpit-Bandejiya) [reply osbenchmark/worker_coordinator/runner.py]: Done,thanks!

**Comment by Arpit-Bandejiya** [comment]
Hi @achitojha, @engechas , please take a look at this PR.
Thanks!

**Review by dhruv16dhr — APPROVED** [review]

File: osbenchmark/worker_coordinator/runner.py [hunk 3]
```diff
@@ -1848,32 +1849,43 @@ def __repr__(self, *args, **kwargs):
...
         id = response.get("id")
         CompositeContext.put(op_name, id)
 
     def __repr__(self, *args, **kwargs):
-        return "open-point-in-time"
+        return "create-point-in-time"
 
 
-class ClosePointInTime(Runner):
+class DeletePointInTime(Runner):
```
Comment (bharath-techie) [inline osbenchmark/worker_coordinator/runner.py:1867]: I remember us discussing to have deleteAllPointInTime outside composite context - lets see if we can add that as well to code repository.

File: osbenchmark/worker_coordinator/runner.py [hunk 4]
```diff
@@ -61,8 +61,9 @@ def register_default_runners():
     register_runner(workload.OperationType.SubmitAsyncSearch, SubmitAsyncSearch(), async_runner=True)
     register_runner(workload.OperationType.GetAsyncSearch, Retry(GetAsyncSearch(), retry_until_success=True), async_runner=True)
     register_runner(workload.OperationType.DeleteAsyncSearch, DeleteAsyncSearch(), async_runner=True)
-    register_runner(workload.OperationType.OpenPointInTime, OpenPointInTime(), async_runner=True)
-    register_runner(workload.OperationType.ClosePointInTime, ClosePointInTime(), async_runner=True)
```
Comment (achitojha) [inline osbenchmark/worker_coordinator/runner.py:65]: Can we confirm none of the [workloads here](https://github.com/opensearch-project/opensearch-benchmark-workloads) do not use these operations

**Review by achitojha — COMMENTED** [review]
Have we tested this with a test workload? I'd recommend having a quick test workload to verify that this is working before we close it

File: osbenchmark/worker_coordinator/runner.py (hunk 4 above)
Comment (Arpit-Bandejiya) [reply osbenchmark/worker_coordinator/runner.py:65]: Yes, I have checked and verified that these operations are not called in the workloads repo as of now

File: tests/worker_coordinator/runner_test.py [hunk 5]
```diff
@@ -4419,30 +4419,69 @@ async def test_can_only_be_run_in_composite(self, opensearch):
...
+
+    @mock.patch("opensearchpy.OpenSearch")
+    @run_async
+    async def test_get_all_point_in_time_in_composite(self, opensearch):
+        pit_id = "0123456789abcdef"
+        params = {}
+        opensearch.list_all_point_in_time.return_value = as_future({
+            "pits": [
+                {
+                    "pitId": pit_id,
```
Comment (bharath-techie) [inline tests/worker_coordinator/runner_test.py:4474]: this i think will be changed to "pit_id" as that is followed everywhere. Lets wait for rest PRs to go in.

File: tests/worker_coordinator/runner_test.py [hunk 6]
```diff
@@ -4403,9 +4403,9 @@ async def test_creates_point_in_time(self, opensearch):
             "index": "test-index"
         }
 
-        opensearch.open_point_in_time.return_value = as_future({"id": pit_id})
+        opensearch.create_point_in_time.return_value = as_future({"id": pit_id})
```
Comment (bharath-techie) [inline tests/worker_coordinator/runner_test.py:4406]: same here , this might become "pit_id" - lets wait for rest prs to merge.

**Review by bharath-techie — APPROVED** [review]

**Comment by IanHoang** [comment]
Just to follow up on Achit's comment, has this been tested on a test workload? Given that the integration tests are failing (but due to a separate cause), it'd be worth testing this on a test workload and supply the results.

**Comment by Arpit-Bandejiya** [comment]
```
(.venv) [ec2-user@ip-172-31-30-121 opensearch-benchmark]$ opensearch-benchmark execute_test --distribution-version=1.0.0 --workload=geonames --test-mode

   ____                  _____                      __       ____                  __                         __
  / __ \____  ___  ____ / ___/___  ____ ___________/ /_     / __ )___  ____  _____/ /_  ____ ___  ____ ______/ /__
 / / / / __ \/ _ \/ __ \\__ \/ _ \/ __ `/ ___/ ___/ __ \   / __  / _ \/ __ \/ ___/ __ \/ __ `__ \/ __ `/ ___/ //_/
/ /_/ / /_/ /  __/ / / /__/ /  __/ /_/ / /  / /__/ / / /  / /_/ /  __/ / / / /__/ / / / / / / / / /_/ / /  / ,<
\____/ .___/\___/_/ /_/____/\___/\__,_/_/   \___/_/ /_/  /_____/\___/_/ /_/\___/_/ /_/_/ /_/ /_/\__,_/_/  /_/|_|
    /_/

[INFO] Preparing for test execution ...
[INFO] Downloading OpenSearch 1.0.0 (411.1 MB total size)                           [100%]
[INFO] Downloading workload data (20.5 kB total size)                             [100.0%]
[INFO] Decompressing workload data from [/home/ec2-user/.benchmark/benchmarks/data/geonames/documents-2-1k.json.bz2] to [/home/ec2-user/.benchmark/benchmarks/data/geonames/documents-2-1k.json] ... [OK]
[INFO] Preparing file offset table for [/home/ec2-user/.benchmark/benchmarks/data/geonames/documents-2-1k.json] ... [OK]
[INFO] Executing test with workload [geonames], test_procedure [append-no-conflicts] and provision_config_instance ['defaults'] with version [1.0.0].

Running delete-index                                                           [100% done]
Running create-index                                                           [100% done]
Running check-cluster-health                                                   [100% done]
Running index-append                                                           [100% done]
Running refresh-after-index                                                    [100% done]
Running force-merge                                                            [100% done]
Running refresh-after-force-merge                                              [100% done]
Running wait-until-merges-finish                                               [100% done]
Running index-stats                                                            [100% done]
Running node-stats                                                             [100% done]
Running default                                                                [100% done]
Running term                                                                   [100% done]
Running phrase                                                                 [100% done]
Running country_agg_uncached                                                   [100% done]
Running country_agg_cached                                                     [100% done]
Running scroll                                                                 [100% done]
Running expression                                                             [100% done]
Running painless_static                                                        [100% done]
Running painless_dynamic                                                       [100% done]
Running decay_geo_gauss_function_score                                         [100% done]
Running decay_geo_gauss_script_score                                           [100% done]
Running field_value_function_score                                             [100% done]
Running field_value_script_score                                               [100% done]
Running large_terms                                                            [100% done]
Running large_filtered_terms                                                   [100% done]
Running large_prohibited_terms                                                 [100% done]
Running desc_sort_population                                                   [100% done]
Running asc_sort_population                                                    [100% done]
Running asc_sort_with_after_population                                         [100% done]
Running desc_sort_geonameid                                                    [100% done]
Running desc_sort_with_after_geonameid                                         [100% done]
Running asc_sort_geonameid                                                     [100% done]
Running asc_sort_with_after_geonameid                                          [100% done]

------------------------------------------------------
    _______             __   _____
   / ____(_)___  ____ _/ /  / ___/_________  ________
  / /_  / / __ \/ __ `/ /   \__ \/ ___/ __ \/ ___/ _ \
 / __/ / / / / / /_/ / /   ___/ / /__/ /_/ / /  /  __/
/_/   /_/_/ /_/\__,_/_/   /____/\___/\____/_/   \___/
------------------------------------------------------

|                                                         Metric |                           Task |       Value |    Unit |
|---------------------------------------------------------------:|-------------------------------:|------------:|--------:|
|                     Cumulative indexing time of primary shards |                                |      0.0337 |     min |
|             Min cumulative indexing time across primary shards |                                |      0.0037 |     min |
|          Median cumulative indexing time across primary shards |                                |  0.00531667 |     min |
|             Max cumulative indexing time across primary shards |                                |   0.0139167 |     min |
|            Cumulative indexing throttle time of primary shards |                                |           0 |     min |
|    Min cumulative indexing throttle time across primary shards |                                |           0 |     min |
| Median cumulative indexing throttle time across primary shards |                                |           0 |     min |
|    Max cumulative indexing throttle time across primary shards |                                |           0 |     min |
|                        Cumulative merge time of primary shards |                                |           0 |     min |
|                       Cumulative merge count of primary shards |                                |           0 |         |
|                Min cumulative merge time across primary shards |                                |           0 |     min |
|             Median cumulative merge time across primary shards |                                |           0 |     min |
|                Max cumulative merge time across primary shards |                                |           0 |     min |
|               Cumulative merge throttle time of primary shards |                                |           0 |     min |
|       Min cumulative merge throttle time across primary shards |                                |           0 |     min |
|    Median cumulative merge throttle time across primary shards |                                |           0 |     min |
|       Max cumulative merge throttle time across primary shards |                                |           0 |     min |
|                      Cumulative refresh time of primary shards |                                |      0.0119 |     min |
|                     Cumulative refresh count of primary shards |                                |          28 |         |
|              Min cumulative refresh time across primary shards |                                |  0.00103333 |     min |
|           Median cumulative refresh time across primary shards |                                |      0.0024 |     min |
|              Max cumulative refresh time across primary shards |                                |  0.00321667 |     min |
|                        Cumulative flush time of primary shards |                                |           0 |     min |
|                       Cumulative flush count of primary shards |                                |           0 |         |
|                Min cumulative flush time across primary shards |                                |           0 |     min |
|             Median cumulative flush time across primary shards |                                |           0 |     min |
|                Max cumulative flush time across primary shards |                                |           0 |     min |
|                                        Total Young Gen GC time |                                |       0.071 |       s |
|                                       Total Young Gen GC count |                                |           2 |         |
|                                          Total Old Gen GC time |                                |       0.054 |       s |
|                                         Total Old Gen GC count |                                |           1 |         |
|                                                     Store size |                                | 0.000446533 |      GB |
|                                                  Translog size |                                | 2.56114e-07 |      GB |
|                                         Heap used for segments |                                |    0.102612 |      MB |
|                                       Heap used for doc values |                                |   0.0168037 |      MB |
|                                            Heap used for terms |                                |   0.0694885 |      MB |
|                                            Heap used for norms |                                |  0.00933838 |      MB |
|                                           Heap used for points |                                |           0 |      MB |
|                                    Heap used for stored fields |                                |   0.0069809 |      MB |
|                                                  Segment count |                                |          15 |         |
|                                                 Min Throughput |                   index-append |     1360.21 |  docs/s |
|                                                Mean Throughput |                   index-append |     1360.21 |  docs/s |
|                                              Median Throughput |                   index-append |     1360.21 |  docs/s |
|                                                 Max Throughput |                   index-append |     1360.21 |  docs/s |
|                                        50th percentile latency |                   index-append |     688.366 |      ms |
|                                       100th percentile latency |                   index-append |     718.436 |      ms |
|                                   50th percentile service time |                   index-append |     688.366 |      ms |
|                                  100th percentile service time |                   index-append |     718.436 |      ms |
|                                                     error rate |                   index-append |           0 |       % |
|                                                 Min Throughput |       wait-until-merges-finish |       53.97 |   ops/s |
|                                                Mean Throughput |       wait-until-merges-finish |       53.97 |   ops/s |
|                                              Median Throughput |       wait-until-merges-finish |       53.97 |   ops/s |
|                                                 Max Throughput |       wait-until-merges-finish |       53.97 |   ops/s |
|                                       100th percentile latency |       wait-until-merges-finish |     17.8691 |      ms |
|                                  100th percentile service time |       wait-until-merges-finish |     17.8691 |      ms |
|                                                     error rate |       wait-until-merges-finish |           0 |       % |
|                                                 Min Throughput |                    index-stats |      119.19 |   ops/s |
|                                                Mean Throughput |                    index-stats |      119.19 |   ops/s |
|                                              Median Throughput |                    index-stats |      119.19 |   ops/s |
|                                                 Max Throughput |                    index-stats |      119.19 |   ops/s |
|                                       100th percentile latency |                    index-stats |     14.1422 |      ms |
|                                  100th percentile service time |                    index-stats |     5.35722 |      ms |
|                                                     error rate |                    index-stats |           0 |       % |
|                                                 Min Throughput |                     node-stats |        85.6 |   ops/s |
|                                                Mean Throughput |                     node-stats |        85.6 |   ops/s |
|                                              Median Throughput |                     node-stats |        85.6 |   ops/s |
|                                                 Max Throughput |                     node-stats |        85.6 |   ops/s |
|                                       100th percentile latency |                     node-stats |     17.0414 |      ms |
|                                  100th percentile service time |                     node-stats |     5.04488 |      ms |
|                                                     error rate |                     node-stats |           0 |       % |
|                                                 Min Throughput |                        default |       13.51 |   ops/s |
|                                                Mean Throughput |                        default |       13.51 |   ops/s |
|                                              Median Throughput |                        default |       13.51 |   ops/s |
|                                                 Max Throughput |                        default |       13.51 |   ops/s |
|                                       100th percentile latency |                        default |     83.4302 |      ms |
|                                  100th percentile service time |                        default |     9.18418 |      ms |
|                                                     error rate |                        default |           0 |       % |
|                                                 Min Throughput |                           term |       74.03 |   ops/s |
|                                                Mean Throughput |                           term |       74.03 |   ops/s |
|                                              Median Throughput |                           term |       74.03 |   ops/s |
|                                                 Max Throughput |                           term |       74.03 |   ops/s |
|                                       100th percentile latency |                           term |     17.8622 |      ms |
|                                  100th percentile service time |                           term |     4.14004 |      ms |
|                                                     error rate |                           term |           0 |       % |
|                                                 Min Throughput |                         phrase |          51 |   ops/s |
|                                                Mean Throughput |                         phrase |          51 |   ops/s |
|                                              Median Throughput |                         phrase |          51 |   ops/s |
|                                                 Max Throughput |                         phrase |          51 |   ops/s |
|                                       100th percentile latency |                         phrase |     24.3141 |      ms |
|                                  100th percentile service time |                         phrase |     4.40979 |      ms |
|                                                     error rate |                         phrase |           0 |       % |
|                                                 Min Throughput |           country_agg_uncached |       13.06 |   ops/s |
|                                                Mean Throughput |           country_agg_uncached |       13.06 |   ops/s |
|                                              Median Throughput |           country_agg_uncached |       13.06 |   ops/s |
|                                                 Max Throughput |           country_agg_uncached |       13.06 |   ops/s |
|                                       100th percentile latency |           country_agg_uncached |     85.2674 |      ms |
|                                  100th percentile service time |           country_agg_uncached |     8.36968 |      ms |
|                                                     error rate |           country_agg_uncached |           0 |       % |
|                                                 Min Throughput |             country_agg_cached |       56.97 |   ops/s |
|                                                Mean Throughput |             country_agg_cached |       56.97 |   ops/s |
|                                              Median Throughput |             country_agg_cached |       56.97 |   ops/s |
|                                                 Max Throughput |             country_agg_cached |       56.97 |   ops/s |
|                                       100th percentile latency |             country_agg_cached |      37.524 |      ms |
|                                  100th percentile service time |             country_agg_cached |     19.6053 |      ms |
|                                                     error rate |             country_agg_cached |           0 |       % |
|                                                 Min Throughput |                         scroll |       19.47 | pages/s |
|                                                Mean Throughput |                         scroll |       19.47 | pages/s |
|                                              Median Throughput |                         scroll |       19.47 | pages/s |
|                                                 Max Throughput |                         scroll |       19.47 | pages/s |
|                                       100th percentile latency |                         scroll |     148.682 |      ms |
|                                  100th percentile service time |                         scroll |     45.4231 |      ms |
|                                                     error rate |                         scroll |           0 |       % |
|                                                 Min Throughput |                     expression |        6.64 |   ops/s |
|                                                Mean Throughput |                     expression |        6.64 |   ops/s |
|                                              Median Throughput |                     expression |        6.64 |   ops/s |
|                                                 Max Throughput |                     expression |        6.64 |   ops/s |
|                                       100th percentile latency |                     expression |     163.061 |      ms |
|                                  100th percentile service time |                     expression |     12.0773 |      ms |
|                                                     error rate |                     expression |           0 |       % |
|                                                 Min Throughput |                painless_static |        3.79 |   ops/s |
|                                                Mean Throughput |                painless_static |        3.79 |   ops/s |
|                                              Median Throughput |                painless_static |        3.79 |   ops/s |
|                                                 Max Throughput |                painless_static |        3.79 |   ops/s |
|                                       100th percentile latency |                painless_static |     281.514 |      ms |
|                                  100th percentile service time |                painless_static |      17.241 |      ms |
|                                                     error rate |                painless_static |           0 |       % |
|                                                 Min Throughput |               painless_dynamic |       10.28 |   ops/s |
|                                                Mean Throughput |               painless_dynamic |       10.28 |   ops/s |
|                                              Median Throughput |               painless_dynamic |       10.28 |   ops/s |
|                                                 Max Throughput |               painless_dynamic |       10.28 |   ops/s |
|                                       100th percentile latency |               painless_dynamic |     107.911 |      ms |
|                                  100th percentile service time |               painless_dynamic |     10.1947 |      ms |
|                                                     error rate |               painless_dynamic |           0 |       % |
|                                                 Min Throughput | decay_geo_gauss_function_score |       52.08 |   ops/s |
|                                                Mean Throughput | decay_geo_gauss_function_score |       52.08 |   ops/s |
|                                              Median Throughput | decay_geo_gauss_function_score |       52.08 |   ops/s |
|                                                 Max Throughput | decay_geo_gauss_function_score |       52.08 |   ops/s |
|                                       100th percentile latency | decay_geo_gauss_function_score |     29.2917 |      ms |
|                                  100th percentile service time | decay_geo_gauss_function_score |     9.72357 |      ms |
|                                                     error rate | decay_geo_gauss_function_score |           0 |       % |
|                                                 Min Throughput |   decay_geo_gauss_script_score |       28.94 |   ops/s |
|                                                Mean Throughput |   decay_geo_gauss_script_score |       28.94 |   ops/s |
|                                              Median Throughput |   decay_geo_gauss_script_score |       28.94 |   ops/s |
|                                                 Max Throughput |   decay_geo_gauss_script_score |       28.94 |   ops/s |
|                                       100th percentile latency |   decay_geo_gauss_script_score |     42.7515 |      ms |
|                                  100th percentile service time |   decay_geo_gauss_script_score |     7.81363 |      ms |
|                                                     error rate |   decay_geo_gauss_script_score |           0 |       % |
|                                                 Min Throughput |     field_value_function_score |       56.65 |   ops/s |
|                                                Mean Throughput |     field_value_function_score |       56.65 |   ops/s |
|                                              Median Throughput |     field_value_function_score |       56.65 |   ops/s |
|                                                 Max Throughput |     field_value_function_score |       56.65 |   ops/s |
|                                       100th percentile latency |     field_value_function_score |     27.6161 |      ms |
|                                  100th percentile service time |     field_value_function_score |     9.54783 |      ms |
|                                                     error rate |     field_value_function_score |           0 |       % |
|                                                 Min Throughput |       field_value_script_score |       25.41 |   ops/s |
|                                                Mean Throughput |       field_value_script_score |       25.41 |   ops/s |
|                                              Median Throughput |       field_value_script_score |       25.41 |   ops/s |
|                                                 Max Throughput |       field_value_script_score |       25.41 |   ops/s |
|                                       100th percentile latency |       field_value_script_score |     51.9738 |      ms |
|                                  100th percentile service time |       field_value_script_score |     12.2132 |      ms |
|                                                     error rate |       field_value_script_score |           0 |       % |
|                                                 Min Throughput |                    large_terms |        0.62 |   ops/s |
|                                                Mean Throughput |                    large_terms |        0.62 |   ops/s |
|                                              Median Throughput |                    large_terms |        0.62 |   ops/s |
|                                                 Max Throughput |                    large_terms |        0.62 |   ops/s |
|                                       100th percentile latency |                    large_terms |     2251.72 |      ms |
|                                  100th percentile service time |                    large_terms |     625.529 |      ms |
|                                                     error rate |                    large_terms |           0 |       % |
|                                                 Min Throughput |           large_filtered_terms |        1.92 |   ops/s |
|                                                Mean Throughput |           large_filtered_terms |        1.92 |   ops/s |
|                                              Median Throughput |           large_filtered_terms |        1.92 |   ops/s |
|                                                 Max Throughput |           large_filtered_terms |        1.92 |   ops/s |
|                                       100th percentile latency |           large_filtered_terms |     756.066 |      ms |
|                                  100th percentile service time |           large_filtered_terms |      226.01 |      ms |
|                                                     error rate |           large_filtered_terms |           0 |       % |
|                                                 Min Throughput |         large_prohibited_terms |        4.24 |   ops/s |
|                                                Mean Throughput |         large_prohibited_terms |        4.24 |   ops/s |
|                                              Median Throughput |         large_prohibited_terms |        4.24 |   ops/s |
|                                                 Max Throughput |         large_prohibited_terms |        4.24 |   ops/s |
|                                       100th percentile latency |         large_prohibited_terms |     457.715 |      ms |
|                                  100th percentile service time |         large_prohibited_terms |     212.425 |      ms |
|                                                     error rate |         large_prohibited_terms |           0 |       % |
|                                                 Min Throughput |           desc_sort_population |       42.58 |   ops/s |
|                                                Mean Throughput |           desc_sort_population |       42.58 |   ops/s |
|                                              Median Throughput |           desc_sort_population |       42.58 |   ops/s |
|                                                 Max Throughput |           desc_sort_population |       42.58 |   ops/s |
|                                       100th percentile latency |           desc_sort_population |     32.0167 |      ms |
|                                  100th percentile service time |           desc_sort_population |     8.15744 |      ms |
|                                                     error rate |           desc_sort_population |           0 |       % |
|                                                 Min Throughput |            asc_sort_population |       69.54 |   ops/s |
|                                                Mean Throughput |            asc_sort_population |       69.54 |   ops/s |
|                                              Median Throughput |            asc_sort_population |       69.54 |   ops/s |
|                                                 Max Throughput |            asc_sort_population |       69.54 |   ops/s |
|                                       100th percentile latency |            asc_sort_population |     20.2246 |      ms |
|                                  100th percentile service time |            asc_sort_population |     5.56675 |      ms |
|                                                     error rate |            asc_sort_population |           0 |       % |
|                                                 Min Throughput | asc_sort_with_after_population |       81.65 |   ops/s |
|                                                Mean Throughput | asc_sort_with_after_population |       81.65 |   ops/s |
|                                              Median Throughput | asc_sort_with_after_population |       81.65 |   ops/s |
|                                                 Max Throughput | asc_sort_with_after_population |       81.65 |   ops/s |
|                                       100th percentile latency | asc_sort_with_after_population |     18.5438 |      ms |
|                                  100th percentile service time | asc_sort_with_after_population |     5.99696 |      ms |
|                                                     error rate | asc_sort_with_after_population |           0 |       % |
|                                                 Min Throughput |            desc_sort_geonameid |       81.92 |   ops/s |
|                                                Mean Throughput |            desc_sort_geonameid |       81.92 |   ops/s |
|                                              Median Throughput |            desc_sort_geonameid |       81.92 |   ops/s |
|                                                 Max Throughput |            desc_sort_geonameid |       81.92 |   ops/s |
|                                       100th percentile latency |            desc_sort_geonameid |     20.1154 |      ms |
|                                  100th percentile service time |            desc_sort_geonameid |     7.57572 |      ms |
|                                                     error rate |            desc_sort_geonameid |           0 |       % |
|                                                 Min Throughput | desc_sort_with_after_geonameid |       78.41 |   ops/s |
|                                                Mean Throughput | desc_sort_with_after_geonameid |       78.41 |   ops/s |
|                                              Median Throughput | desc_sort_with_after_geonameid |       78.41 |   ops/s |
|                                                 Max Throughput | desc_sort_with_after_geonameid |       78.41 |   ops/s |
|                                       100th percentile latency | desc_sort_with_after_geonameid |       19.21 |      ms |
|                                  100th percentile service time | desc_sort_with_after_geonameid |     6.15332 |      ms |
|                                                     error rate | desc_sort_with_after_geonameid |           0 |       % |
|                                                 Min Throughput |             asc_sort_geonameid |       75.05 |   ops/s |
|                                                Mean Throughput |             asc_sort_geonameid |       75.05 |   ops/s |
|                                              Median Throughput |             asc_sort_geonameid |       75.05 |   ops/s |
|                                                 Max Throughput |             asc_sort_geonameid |       75.05 |   ops/s |
|                                       100th percentile latency |             asc_sort_geonameid |     22.2586 |      ms |
|                                  100th percentile service time |             asc_sort_geonameid |      8.5623 |      ms |
|                                                     error rate |             asc_sort_geonameid |           0 |       % |
|                                                 Min Throughput |  asc_sort_with_after_geonameid |       95.37 |   ops/s |
|                                                Mean Throughput |  asc_sort_with_after_geonameid |       95.37 |   ops/s |
|                                              Median Throughput |  asc_sort_with_after_geonameid |       95.37 |   ops/s |
|                                                 Max Throughput |  asc_sort_with_after_geonameid |       95.37 |   ops/s |
|                                       100th percentile latency |  asc_sort_with_after_geonameid |     15.0555 |      ms |
|                                  100th percentile service time |  asc_sort_with_after_geonameid |     4.36249 |      ms |
|                                                     error rate |  asc_sort_with_after_geonameid |           0 |       % |


--------------------------------
[INFO] SUCCESS (took 45 seconds)

```

**Comment by Arpit-Bandejiya** [comment]
> Just to follow up on Achit's comment, has this been tested on a test workload? Given that the integration tests are failing (but due to a separate cause), it'd be worth testing this on a test workload and supply the results.

Thanks @IanHoang for the feedback, have uploaded the test results. Also the `create-point-in-time`, `delete-point-in-time` and `list-all-point-in-time` are not present in any default workloads. Thanks!

**Review by IanHoang — APPROVED** [review]
Just to reiterate, OSB results report does not include `create-point-in-time`, `delete-point-in-time`, and `list-all-point-in-time` because they are not present in any current public OSB workloads.

**Review by travisbenedict — APPROVED** [review]

**Pull request merged**